appeals/api/src/server/endpoints/documents/documents.service.js

```diff
@@ -33,12 +52,143 @@ export const getDocumentsForAppeal = async (appealId, sectionName = null) => {
 	return folderLayout;
 };
 
+/**
+ * Returns a list of document paths available for the current Appeal
+ * @param {string} documentId
+ * @returns {Promise<Document>}
+ */
+export const getDocumentForAppeal = async (documentId) => {
+	const document = await getDocumentById(documentId);
+	return document;
+};
+
+/**
+ * @param {DocumentApiRequest} upload
+ * @param {number} appealId
+ * @returns {Promise<{documents: BlobInfo[]}>}}
+ */
+export const addDocumentsToAppeal = async (upload, appealId) => {
+	const appeal = await getAppeal(appealId);
```

Comment:
There is a middleware function that can be added to a route which uses the `appealId` param and checks that the appeal exists and if so then adds the appeal to the request so it can be used later on.

By using this you shouldn't then need to have an additional method to fetch the appeal separately as you'll already have those details available.